test/libsolidity/syntaxTests/calldata_struct_argument_with_internal_type_inside.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+  struct S {
+    function() a;
+  }
+  function f(S[2] calldata) {}
+}
+// ----
+// SyntaxError 4937: (50-78): No visibility specified. Did you intend to add "public"?
```

Comment:
Please make the function public (`function <functionName>() public ...`) so that this error goes away - it is not relevant for the test.